src/lib/genesis_ledger_helper/genesis_ledger_helper.ml

```diff
@@ -933,20 +931,24 @@ module Config_loader : Config_loader_intf = struct
     let values = Genesis_proof.create_values_no_proof inputs in
     (values, config)
 
-  let load_config_files ~logger ~genesis_constants ~constraint_constants
-      ~conf_dir ~genesis_dir ~cli_proof_level ~proof_level
-      (config_files : string list) =
+  let load_config_files ?overwrite_version ?genesis_dir ?(itn_features = false)
+      ?cli_proof_level ?conf_dir ~logger (config_files : string list) =
     let open Deferred.Or_error.Let_syntax in
     let genesis_dir =
+      let%map.Option conf_dir = conf_dir in
       Option.value ~default:(conf_dir ^/ "genesis") genesis_dir
     in
+    let%bind.Deferred constants =
+      Runtime_config.Constants.load_constants ?conf_dir ?cli_proof_level
+        ~itn_features ~logger config_files
+    in
     let%bind config =
-      Runtime_config.Json_loader.load_config_files ~conf_dir ~logger
+      Runtime_config.Json_loader.load_config_files ?conf_dir ~logger
```

Comment:
Logging behavior doesn't change comparing to previous version, which was present in `mina_cli_entrypoint`